tests/resource_filtering_test.go

```diff
@@ -1368,25 +1354,20 @@ var _ = Describe("Resource includes", func() {
 				Expect(err).ToNot(HaveOccurred())
 
 				By("Veryfing backup")
-				backup, err := framework.GetBackup(timeout, backupName, f.BackupNamespace)
-				Expect(err).ToNot(HaveOccurred())
-				Expect(backup.Status.Progress.ItemsBackedUp).To(Equal(backup.Status.Progress.TotalItems))
-
-				// The backup should contain the following 12 items:
-				// - VirtualMachine
-				// - 2 DataVolume
-				// - 2 PVC
-				// - 2 PV
-				// - 2 VolumeSnapshot
-				// - 2 VolumeSnapshotContent
-				// - VolumeSpapshotClass
-				// - DataVolume CRD
-				// - Namespace
-				expectedItems := 14
-				if framework.IsDataVolumeGC(f.KvClient) {
-					expectedItems -= 2
+				expectedItems := map[string][]string{
+					"CustomResourceDefinition": []string{"virtualmachines"},
```

Comment:
Not necessary if the VM exists

Reply:
it is possible velero might introduce some bug that will backup the vm but not the crd, better have the check. same for dv